modules/core/04-channel/keeper/timeout.go

```diff
@@ -83,10 +83,10 @@ func (k Keeper) TimeoutPacket(
 		return types.ErrNoOpMsg
 	}
 
-	if channel.State != types.OPEN {
+	if channel.State != types.OPEN && channel.FlushStatus != types.FLUSHING {
```

Comment:
rejects all channels that aren't OPEN and that are not currently flushing (which leaves us with `TRYUPGRADE` and `ACKUPGRADE` in a `FLUSHING` state allowed to timeout packets).